### PR TITLE
Youtube page

### PR DESCRIPTION
**Pull request by El-Potato-Slayer**

### CODE REVIEW

**Comment by peterrobert** [comment]
### PULL REQUEST UNDER REVIEW BY PETER ROBERT (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    <main class="grid-container">
+        <div id="video-section">
+            <iframe width="854" height="480" src="https://www.youtube.com/embed/Zo5YRdRh3PY"></iframe>
+            <h3 id="title">Fall in Archaea -- Motive</h3>
+            <div class="grid-container" id="video-info">
+            <span class="small gray">2,000 views June 4, 2020</span>
+                <div>
+                    <i class="fa fa-thumbs-up fa-lg gray"><span class="small pad-left-8">3.8M</span></i>
+                    <i class="fa fa-thumbs-down fa-lg gray"><span class="small pad-left-8">2.5K</span></i>
+                    <i class="fa fa-share fa-lg gray"><span class="small pad-left-8">Share</span></i>
```
Comment (peterrobert) [inline index.html]: Kindly make this a button. It should be a button, not an icon like described:arrow_down: 
share_btn

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    grid-template-columns: 1fr 1fr;
+    /* margin-top: 20px; */
+    margin: 24px 34px;
+}
+
+/* Video-section */
+
+#video-section{
+    /* margin-left: 100px; */
+    /* margin-top: 20px; */
```
Comment (peterrobert) [inline styles.css]: Kindly remove all commented code its a requirement at microverse since it keeps your code clean

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+#right-nav{
+    justify-self: end;
+}
+
+/* Main section */
+
+main{
+    grid-template-columns: 1fr 1fr;
+    /* margin-top: 20px; */
```
Comment (peterrobert) [inline styles.css]: Remove commented code

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: 
Great project:+1:  very impressive work :+1: However there are a few reasons that have prevented me from approving this project. Kindly make these changes :arrow_down: 

- [x] The readme file looks good but let's make it even better :smiley:  Kindly make your readme file more descriptive by using this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/l) as a reference 

### very good project. 
### HAPPY CODING :computer: 

**If you have any question kindly reach me on** @peterrobert

File: index.html (hunk 1 above)
Comment (El-Potato-Slayer) [reply index.html]: Thank you for the review. I've changed icon into a button

**Comment by peterrobert** [comment]
### PULL REQUEST UNDER REVIEW BY PETER ROBERT

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: 
you have done a good job but still, you didn't add more information to your readme file

Kindly do that and I will approve your project

**Comment by El-Potato-Slayer** [comment]
Could you please specify what information needs to be added to the readme file? Do I need a more detailed description or am I missing certain parts?

**Comment by voscarmv** [comment]
# Pull Request Under Review by Oscar Mier (TSE) 👨‍💻 🐦

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+button:hover{
+    cursor: pointer;
+}
+
+/* Navigation bar */
```
Comment (voscarmv) [inline styles.css:15]: Please remove all commented code from your `CSS` file!

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Great progress team! 💪 🧑‍💻 
## Please implement the following changes before continuing 👀 
- Your `README.md` file looks nice. However, it's missing a couple of sections from the suggested Microverse template [here](https://github.com/microverseinc/readme-template).
- Consider adding an [Open-Source License](https://en.wikipedia.org/wiki/Open-source_license) to your project and referencing it on your `README.md`. This will make your project look more professional to potential employers browsing GitHub for talent! 👨‍💼 👩‍💼 
- It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) 
image
### Implement these changes and then request a re-review! 😄 👍 
### Comments or questions? Contact me on [Twitter](http://www.twitter.com/voscarmv) or [LinkedIn](http://www.linkedin.com/in/oscar-mier) 🧑‍💻 🗨️ 🧑‍💻 
## Happy hacking! 🧠 🖱️ ⌨️ 🖥️

File: styles.css (hunk 4 above)
Comment (El-Potato-Slayer) [reply styles.css:15]: Hi Oscar. From what I've read before I started the Main Curriculum is that these comments are needed to provide structure and allow ease of reading the code. The comments that make the code messy are code that are commented. Once I find the article, I will send you the link. And thank you for the rest of the feedback.

Comment (El-Potato-Slayer) [reply styles.css:15]: Here is the link: https://speckyboy.com/good-bad-css-practices/
In the "Good practices" section, it mentions making use of a table of content.

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

**Review by andresporras3423 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
**Hello @El-Potato-Slayer and @Arinpe** 
Amazing job :clap: :clap: you have passed all the requirements for this project
Now you are ready to continue with your next challenge :muscle: 

#### OPTIONAL

- [ ] I encourage you to use <footer> tag for the footer
- [ ] Try not to use multiple ID selectors in your CSS file

Don't forget to merge with the master branch

Keep going like this,

Oscar Andrés Russi Porras

**Comment by El-Potato-Slayer** [comment]
Hi Andres. My partner's github is: https://github.com/GraceOyiza/ I'm not sure who @Arinpe is.

**Comment by Arinpe** [comment]
Hi,
I didn't do any project. As a matter of fact I have not been given a start
up date yet. I have no idea who El-Potato -Slayer is.
Thanks.

**Comment by El-Potato-Slayer** [comment]
> I have no idea who El-Potato -Slayer is.

I am the apocalypse that all potatoes fear. My name instills horror within the very core of their souls.

**Comment by andresporras3423** [comment]
Hi guys, my apologies. @Arinpe appears as one of the contributors in the commit history. Make sure that you are using the same email in your local git config credentials.

**Pull request merged**